LICENSE-3rdparty.csv

```diff
@@ -1,888 +1,83 @@
 Component,Origin,License
-core,"github.com/frapposelli/wwhrd",MIT
```

Comment:
There is a significant diff in the licenses, are we not losing coverage?

Reply:
no, I forgot to add a comment. 

it is only because the `go-licenses` program only the package really used by the agent, not was is vendor

(btw: I'm planning another PR to remove the `vendoring` logic in the makefile)